docs/evolutionaryAlgorithm/README.md

```diff
@@ -0,0 +1,269 @@
+? Please select a data source for import (Use arrow keys)
+❯ villeray 
+  --New data source--
+Running import in dir runtime/imports/villeray/
+Enhancing and saving polygon geojson data to file:  runtime/imports/villeray/osmPolygonsEnhanced.geojson
+Fetch buildings from osm geojson data...
+There are 16103 buildings in the osm geojson data.
+Fetch polygons and multipolygons from osm geojson data...
+=== validated and enhanced polygon 16970 ===
+Finished with 16971 polygons.
+```
+
+3. Import and process osm data to files. This will create 3 geojson files, one for points of interest, one for residential buildings and one for residential zones. This task will give a number of warnings on buildings and places that may have missing or incomplete data and for which an approximation is used. It is worth running this task in non-interactive mode and pipe the output to a file for later processing
+
+```
+yarn node packages/chaire-lib-backend/lib/scripts/osm/2_importAndProcessOsmDataToFiles.task.js --dataSourceId villeray > outputFile.txt
+```
+
+4. (optional) Assign weights to POIs. This will use the "Trip Generation Manual"'s data to give weight to POIs according to their category and the area occupied.
+
+```
+yarn node packages/chaire-lib-backend/lib/scripts/osm/2b_assignWeightsToPOIs.task.js
+```
+
+5. (optional?) Import and validate land role data. This will validate that the number of residences (flats) from openstreetmap matches the land role data and give warnings if it is not the case.
+
+```
+yarn node packages/chaire-lib-backend/lib/scripts/osm/3_importAndValidateLandRoleData.task.js
+```
+
+6. Then import the geojson places files into the database using the places import script by running the following script, which will add 2 places data sources, one for POIs and one for residential entrances.
+
+```
+yarn node packages/transition-backend/lib/scripts/osm/4_importPlacesToDb.task.js
+```
+
+The simulation then takes points from the datasource and calculates the other accessible places from this location within the simulation parameters. Points can have weights (for example, supermarket can have higher weight than local bakery) and their weight is divided by the time to reach this location (places reachable in 10 minutes will have higher weights than those accessible in 60 minutes).
+
+## Run the simulation
+
+Running the simulation is currently done via the command line, so access to the Transition server is required, it cannot be done entirely from the UI.
+
+The command to run for the simulations is as follows, to select the simulation to run:
+
+```
+yarn node packages/transition-backend/lib/scripts/simulations/simulation.task.js
+? Please select the simulation to run (Use arrow keys)
+❯ Test (Genetic algorithm with lines and number of vehicles)
+```
+
+It will display the simulation configuration and invite you to edit them for this specific run, just clicking enter should be good if the parameters are correct. 
```

Comment:
Those are the setting you mention earlier in the screenshot??

Reply:
> Those are the setting you mention earlier in the screenshot??

Exactly, I'll make that clear in the text